aws/rds/postgresql/vars.tf

```diff
@@ -0,0 +1,88 @@
+/*
+ * Required variables
+ */
+variable "app_name" {
+  type = "string"
+}
+
+variable "app_env" {
+  type = "string"
+}
+
+variable "db_name" {
+  type = "string"
+}
+
+variable "db_root_user" {
+  type = "string"
+}
+
+variable "db_root_pass" {
+  type = "string"
+}
+
+variable "subnet_group_name" {
+  type = "string"
+}
+
+variable "availability_zone" {
+  type    = "string"
+  default = ""
```

Comment:
If you want this to be a required variable, you probably don't want to provide a `default` here.